mod-finance/examples/encumbrance.sample

```diff
@@ -1,10 +1,6 @@
 {
-  "id": "177abecb-54c8-4938-9950-ec0a58e1999c",
-  "amountEncumbered": 12.50,
+  "amountExpended": 17.92,
```

Comment:
I think encumbrance sample  should be part of sample for transactions.

Reply:
Thank you pointing on that. Sure, this sample is not required here anymore